### ISSUE TITLE
plugin slows down computer when enable full null support

### ISSUE THREAD

**Issue by yuxinli915** [cfmleditor/cfmleditor#18]
Hi,

Here is my environment: 
- Mac with Apple M chip
- MacOS 14.4.1 (23E224) 
- VScode

When I open a project that have full null support in the server admin. And `this.nullSupport = true;` under `app.cfm`.
The whole computer starts to slow down and eventually got stuck on everything.

**Comment by ghedwards** [comment]
@yuxinli915 as far as I can tell there is no specific change in behavior in this extension when nullSupport is enabled, perhaps some further diagnosis on your end about which process is running away on your Mac, and / or which processes or extensions might be causing this ? It could possibly be the cfmleditor-lint extension or CFLint as well ? Maybe disable all extensions and enable one at a time until you find the culprit ?

**Comment by yuxinli915** [comment]
@ghedwards Thanks for the reply. I did try enabling plugins one by one to see which one is causing the issue. Since the linter is depending on the editor, I tried with the two enabled together and the issue started. Then I tried disable the linter first and the issue still persisted. It was gone when I disabled the editor plugin. For all other plugins, they have no impact on the performance. One more thing is that I have a few projects in CFML and the only difference I can tell is that the one uses the null value is the only one I am having issue with. I would like to do further diagnosis. would you mind to pointing out where I should look for? The activity monitor or VScode console? or anything else. Thanks.

**Comment by ghedwards** [comment]
@yuxinli915 try disabling CFML Signature support, and let me know if that makes any difference ?

<img width="289" alt="image" src="https://github.com/cfmleditor/cfmleditor/assets/125586/5fa3ec11-5146-471f-9263-0ffe8c3e9245">

**Comment by yuxinli915** [comment]
@ghedwards No it didn't help.

**Comment by ghedwards** [comment]
@yuxinli915  I don't have a lot of time to dedicate to this project but I'd love to try help find the issue you're experiencing, if you're on cfml slack please reach out, otherwise maybe check the VScode extension host output or developer tools for any clues ?

**Comment by ghedwards** [comment]
I don't believe there is anything specific in cfmleditor around null support so maybe a red herring, closing unless more information is made available.
